docs/modules/operation/pages/service-assurance/monitors/MemcachedMonitor.adoc

```diff
@@ -87,7 +87,7 @@ The following metrics are collected:
 | Default
 
 | retry
-| Number of attempts to establish the Memcached connnection.
+| Number of attempts to establish the Memcached connection.
```

Comment:
Will be resolved via https://github.com/OpenNMS/opennms/pull/3624

Reply:
I've removed this change from the PR.